api/src/main/java/com/example/api/dto/response/user/dashboard/GeneralStats.java

```diff
@@ -0,0 +1,17 @@
+package com.example.api.dto.response.user.dashboard;
+
+import lombok.*;
+
+@Data
+@AllArgsConstructor
+@Getter
+@Setter
+public class GeneralStats {
+    private Double avgGraphTask;
+    private Double avgCombatTask;
```

Comment:
jak nazywamy wszędzie FileTask to tu też nazwij, jak chcesz żeby w jsonie nazywało sie inaczej nazywało to dajesz @JsonProperty("avgCombatTask"), ale wsm nie wiem po co zawsze używaliśmy FileTask po co zmieniać